### PR TITLE
Updated the structure of the Web App.

### PR DESCRIPTION
**Pull request by Glavin001**
See discussion at Issue #2.

---
- [x] [yeoman/generator-webapp](https://github.com/yeoman/generator-webapp) to setup Web App structure
- [x] [Glavin001/generator-gitignore](https://github.com/Glavin001/generator-gitignore) to add ignore rules for Mac, Linux, Windows, Vim, SublimeText, and Node.js
- [x] Updated UI of the main `index.html` page and other such changes.

---
### For Development

``` bash
grunt serve
```
### For Building Static Web App

We should host on [GitHub Pages](http://pages.github.com/).

``` bash
grunt build
```

### CODE REVIEW

**Comment by vasilionjea** [comment]
Quick recommendation so the link text doesn't overflow (_I forgot to add this_): 

```
#shared-links a {
  word-break: break-all;
}
```

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# ===== Start Node.gitignore =====
+lib-cov
+lcov.info
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
```
Comment (sindresorhus) [inline .gitignore]: only project specific stuff should be in this gitignore, the rest should be in your own global one.

**Comment by Glavin001** [comment]
@sindresorhus, I have removed the unnecessary rules from the GitIgnore. 

---

> Quick recommendation so the link text doesn't overflow (I forgot to add this):

@vasilionjea, I think the UI will change quite a bit from here onwards, so I left out that change from this. For the record, it looked pretty good as is with your styles on my screen :+1:.

**Comment by sindresorhus** [comment]
Awesome! Landed :)

**Comment by Glavin001** [comment]
Thanks! Should we be hosting with [GitHub Pages](http://pages.github.com/)? However, the GitIgnore rules will not push up the `dist` directory after running `grunt build`. 
What're your thoughts on this?

**Comment by sindresorhus** [comment]
Yes, GitHub Pages. I'll get something set up.

**Pull request closed without merging**